gradle.properties

```diff
@@ -8,4 +8,4 @@ APPLICATION_NAME=Amazon DocumentDB JDBC Driver
 # Comment out if not creating a beta version
 #BETA_VERSION=1
 # Uncomment out next line to disable remote integration tests
-#runRemoteIntegrationTests=false
+runRemoteIntegrationTests=false
```

Comment:
I think remote tests should still be run on GitHub?

Reply:
Oops. Thanks.